tools/batch/run_batch_squad.sh

```diff
@@ -12,11 +18,11 @@ for MODEL_NAME in albert_base \
 do
   python3 submit-job.py \
       --region us-east-1 \
-      --source-ref master \
+      --source-ref fix_docker \
       --job-type g4dn.12x \
       --save-path temp \
       --name test_squad2_${MODEL_NAME} \
       --work-dir scripts/question_answering \
-      --remote https://github.com/dmlc/gluon-nlp/ \
-      --command 'bash commands/run_squad2_'${MODEL_NAME}'.sh | tee stdout.log' >> submit_squad_v2.log
+      --remote https://github.com/sxjscience/gluon-nlp/ \
```

Comment:
you may use `master` branch from `dmlc`?

Reply:
Yes, I'm testing the horovod support in AWS Batch. I found some issues and is solving that.

Reply:
The quick update is that the AWS Batch now supports horovod training. I'll update the code accordingly.